internal/pkg/describe/worker_service.go

```diff
@@ -0,0 +1,188 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package describe
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"text/tabwriter"
+
+	cfnstack "github.com/aws/copilot-cli/internal/pkg/deploy/cloudformation/stack"
+	"github.com/aws/copilot-cli/internal/pkg/describe/stack"
+	"github.com/aws/copilot-cli/internal/pkg/manifest"
+	"github.com/aws/copilot-cli/internal/pkg/term/color"
+)
+
+// WorkerServiceDescriber retrieves information about a worker service.
+type WorkerServiceDescriber struct {
+	app             string
+	svc             string
+	enableResources bool
+
+	store                DeployedEnvServicesLister
+	svcDescriber         map[string]ecsSvcDescriber
+	initServiceDescriber func(string) error
+}
+
+// NewWorkerServiceConfig contains fields that initiates WorkerServiceDescriber struct.
+type NewWorkerServiceConfig struct {
+	NewServiceConfig
+	EnableResources bool
+	DeployStore     DeployedEnvServicesLister
+}
+
+// NewWorkerServiceDescriber instantiates a worker service describer.
+func NewWorkerServiceDescriber(opt NewWorkerServiceConfig) (*WorkerServiceDescriber, error) {
```

Comment:
What do you think of this proposal? 

🤔 `svc show` for a worker service seems to be the exact same `svc show` for a backend or lbws without the service discovery and port information.

This means I think we can refactor the ECS service describers this way:
```go
type ecsServiceDescriber struct { // we would need to rename the existing interface to ecsService
	app             string
	svc             string
	enableResources bool

	store                DeployedEnvServicesLister
	svcDescriber         map[string]ecsSvcDescriber
	initServiceDescriber func(string) error
}
func (d *ecsServiceDescriber) Describe() (HumanJSONStringer, error) { ... }

type BackendServiceDescriber {
	*ecsServiceDescriber // embeded

	envDescriber   map[string]envDescriber
}

type LBWebServiceDescriber struct {
	*ecsServiceDescriber // embeded

	envDescriber  map[string]envDescriber
	svcParams map[string]string
}

type WorkerServiceDescriber struct {
	*ecsServiceDescriber // embeded
}
```
This way we don't have to do any work for the `WorkerServiceDescriber` we would get it for free.

If that makes sense I think it might be worthwhile creating a separate refactor PR